var/spack/repos/builtin/packages/intel-oneapi-ccl/package.py

```diff
@@ -18,6 +18,10 @@ class IntelOneapiCcl(IntelOneApiLibraryPackage):
     depends_on('intel-oneapi-mpi')
 
     if platform.system() == 'Linux':
+        version('2021.5.1',
+                url='https://registrationcenter-download.intel.com/akdlm/irc_nas/18472/l_oneapi_ccl_p_2021.5.1.494_offline.sh',
+                sha256='237f45d3c43447460e36eb7d68ae3bf611aa282015e57c7fe06c2004d368a68e',
+                expand=False)
```

Comment:
```suggestion
                expand=False)
                
```

add blank line to change hash and trigger gitlab retest